### PR TITLE
Feature/migrate to jhipster 6.5.x

### PR DESCRIPTION
**Pull request by kkyrychenko**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     "node_modules"
   ],
   "devDependencies": {
-    "generator-jhipster": "5.3.4"
+    "generator-jhipster": "6.5.1",
+    @openapitools/openapi-generator-cli": "0.0.14-4.0.2"
```
Comment (amedvedchuk) [inline package.json]: please check if Json is correct here (`@openapitools` does not start from the quote)

File: src/main/java/com/icthh/xm/tmf/ms/activation/events/bindings/MessagingConfiguration.java [hunk 2]
```diff
@@ -122,7 +125,8 @@ private synchronized void createHandler(String chanelName, String consumerGroup,
             props.getConsumer().setAutoCommitOffset(false);
             props.getConsumer().setAutoCommitOnError(false);
             props.getConsumer().setStartOffset(startOffset);
-            kafkaExtendedBindingProperties.getBindings().put(chanelName, props);
+            System.out.println("Bindings = " + kafkaExtendedBindingProperties.getBindings());
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/events/bindings/MessagingConfiguration.java]: Use logger instead of sysout

Comment (kkyrychenko) [reply src/main/java/com/icthh/xm/tmf/ms/activation/events/bindings/MessagingConfiguration.java]: +

File: package.json (hunk 1 above)
Comment (kkyrychenko) [reply package.json]: fixed

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/SagaTransactionResource.java [hunk 3]
```diff
@@ -32,13 +33,15 @@
...
     @PostMapping("/transaction")
+    @PrivilegeDescription("Privilege to create a new saga transaction")
     public ResponseEntity<SagaTransaction> createSagaTransaction(@RequestBody SagaTransaction sagaTransaction) {
         return ResponseEntity.ok(sagaService.createNewSaga(sagaTransaction));
     }
 
     @Timed
     @PreAuthorize("hasPermission({'id': #id}, 'ACTIVATION.TRANSACTION.CONTINUE')")
     @PostMapping("/task/{id}/continue")
+    @PrivilegeDescription("Privilege to continue task")
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/SagaTransactionResource.java]: suspended task

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/SagaTransactionResource.java [hunk 4]
```diff
@@ -79,12 +86,14 @@
...
     @GetMapping("/transactions/{id}/events")
     @PreAuthorize("hasPermission({'id': #id, 'eventId': #eventId}, 'ACTIVATION.TRANSACTION.GET_EVENTS')")
+    @PrivilegeDescription("Privilege to get all events by transaction")
     public ResponseEntity<List<SagaEvent>> getEventsByTransaction(@PathVariable("id") String id) {
         return ResponseEntity.ok(sagaService.getEventsByTransaction(id));
     }
 
     @GetMapping("/transactions/{id}/logs")
     @PreAuthorize("hasPermission({'id': #id, 'eventId': #eventId}, 'ACTIVATION.TRANSACTION.GET_LOGS')")
+    @PrivilegeDescription("Privilege to get all logs by transaction")
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/SagaTransactionResource.java]: saga log records

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java [hunk 5]
```diff
@@ -21,32 +22,37 @@
 
     @Override
     @PreAuthorize("hasPermission({'tenant':#tenant}, 'ACTIVATION.TENANT.CREATE')")
+    @PrivilegeDescription("Privilege to add a new activation tenant")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java]: suggest description:
`Privilege to add a new tenant on activation micro-service`

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java [hunk 6]
```diff
@@ -21,32 +22,37 @@
...
     public ResponseEntity<Void> addTenant(Tenant tenant) {
         tenantService.createTenant(tenant.getTenantKey());
         return ResponseEntity.ok().build();
     }
 
     @Override
     @PreAuthorize("hasPermission({'tenantKey':#tenantKey}, 'ACTIVATION.TENANT.DELETE')")
+    @PrivilegeDescription("Privilege to delete activation tenant")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java]: suggest description:
`Privilege to delete tenant on activation micro-service`

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java [hunk 7]
```diff
@@ -21,32 +22,37 @@
...
     public ResponseEntity<Void> deleteTenant(String tenantKey) {
         tenantService.deleteTenant(tenantKey);
         return ResponseEntity.ok().build();
     }
 
     @Override
     @PostAuthorize("hasPermission(null, 'ACTIVATION.TENANT.GET_LIST')")
+    @PrivilegeDescription("Privilege to get all activation tenants")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java]: suggest description:
`Privilege to get all tenants on activation micro-service`

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java [hunk 8]
```diff
@@ -21,32 +22,37 @@
...
     public ResponseEntity<List<Tenant>> getAllTenantInfo() {
         throw new UnsupportedOperationException();
     }
 
     @Override
     @PostAuthorize("hasPermission({'returnObject': returnObject.body}, 'ACTIVATION.TENANT.GET_LIST.ITEM')")
+    @PrivilegeDescription("Privilege to get activation tenant")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java]: suggest description:
`Privilege to get tenant by id on activation micro-service`

File: src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java [hunk 9]
```diff
@@ -21,32 +22,37 @@
...
     public ResponseEntity<Tenant> getTenant(String s) {
         throw new UnsupportedOperationException();
     }
 
     @Override
     @PreAuthorize("hasPermission({'tenant':#tenant, 'state':#state}, 'ACTIVATION.TENANT.UPDATE')")
+    @PrivilegeDescription("Privilege to update activation tenant")
```
Comment (amedvedchuk) [inline src/main/java/com/icthh/xm/tmf/ms/activation/web/rest/TenantResource.java]: suggest description:
`Privilege to update tenant on activation micro-service`

**Pull request merged**